### ISSUE TITLE
external package import with '~' is broken

### ISSUE THREAD

**Issue by petergaal91** [ionic-team/stencil-sass#32]
I found a weird bug around the core resolveModule implementation. I'm not sure this is the right place for this, but it's related to this package so I open a bugticket here.

The issue is the following:
We use font-awesome free package in our project and we cannot import the SCSS files directly with `~` for some reason in stencil.

```
@import '~bootstrap/scss/bootstrap'; // <- working
@import '~@fortawesome/fontawesome-free/scss/variables' // <- not working
@import '../../node_modules/@fortawesome/fontawesome-free/scss/variables' // <- working
```

Repository for this bug: https://github.com/petergaal91/stencil-sass-path-resolve-bug

**Comment by petergaal91** [comment]
Ok, I found a solution:

* I added a new package into our project: [node-sass-package-importer](https://www.npmjs.com/package/node-sass-package-importer)
* I updated the stencil.config.ts file with the following:
```
...
import { Config } from '@stencil/core';
import { sass } from '@stencil/sass';
import nodeSassPackageImporter from 'node-sass-package-importer';

export const baseConfig: Config = {
	plugins: [sass({ importer: nodeSassPackageImporter() }]
};
```

I see [here](https://github.com/ionic-team/stencil-sass/blob/1ef603c7114f5756bded051d2ce2d6c1f79447cf/src/util.ts#L72) a custom importer implementation but it's not included or not working properly. I keep this ticket as it is, please fix it.

Thank You!

**Comment by petergaal91** [comment]
Fixed in 1.3.2 version: https://github.com/ionic-team/stencil-sass/commit/47836b5c5695763de47bfe44691394c5c1978eff

**Comment by kirakishin** [comment]
hello,
seems to be back again with :
```
"@stencil/core": "^2.6.0"
"@stencil/sass": "^1.4.1",
```
